paper/paper.md

```diff
@@ -0,0 +1,171 @@
+    affiliation: 1
+  - name: Marjan Albooyeh
+    orcid: 0009-0001-9565-3076
+    equal-contrib: true
+    affiliation: 1
+  - name: Rainier Barrett
+    orcid: 0000-0002-5728-9074
+    equal-contrib: false
+    affiliation: 1
+  - name: Eric Jankowski
+    orcid: 0000-0002-3267-1410
+    corresponding: true
+    affiliation: 1
+affiliations:
+ - name: Boise State University, Boise, ID, USA
+   index: 1
+date: 01 January 2001
+bibliography: paper.bib
+---
+# Summary
+`JankFlow` is a package for reproducibly performing complex HOOMD-blue [@hoomd_2019]
+simulation workflows. It enables the programmatic specification of tasks including
+definition of molecular structures, forcefield definition and application and chaining
+together simulation stages (e.g., shrinking, equilibration, simulating a sequence
+of ensembles, tensile testing) through an extensible set of Python classes.
+The modular design supports a library of complex workflows for organic
+macrmolecular and polymer simulations. Tutorials are provided to demonstrate
+package features and flexibility.
+
+
+# Statement of need
+
+High-level programmatic specification of molecular simulation workflows are
+needed for two reasons: First: They provide the information necessary for a
+simulation study to be reproducible, and Second: They minimize the cognitive
+load of getting started with running experiments [?].
+For a researcher new to molecular simulations, building the necessary set
+of computational tools needed to actually perform experiments simultaneously:
+(a) requires skills and knowledge different from those needed to do research,
+(b) involves repeating work that others have already done.
+
+This is a well recognized problem, and recent advances in well-documented
+open-source tools have made the programmatic specification of
+molecular simulation components easier than ever
+[@hoomd_2019, @lammps_2022, @gromacs_2015, @mbuild_2016, @gmso,
+@Santana-Bonilla_2023, @polyply_2022, @biosimspace_2019].
+Individually, each of these tools lower the cognitive load of one aspect of an
+overall workflow such as representing molecules, building initial structures,
+parameterizing and applying a forcefield, to running simulations.
+However, the challenge of stitching the pieces together to create a complete
+workflow still contains several barriers.
+
+The computational researcher who follows best practices for accurate,
+accessible and reproducible results may create a programmatic layer over these
+individual software packages (i.e. wrapper) that serves to consolidate and
+automate a complete workflow. However, these efforts often use a bespoke approach
+where the entire workflow design is tailored towards the specific question or
+project. Design choices might include the materials studied, the model used
+(e.g. atomistic or coarse-grained), the source of the forcefield in the model, and
+the simulation protocols followed. As a result, this wrapper is likely unusable
```

Comment:
We mentioned "programmatic layer" at the top and then "wrapper" here. Is this clear enought?

Reply:
We have the "(i.e. wrapper)" after "programmatic layer" which I think clarifies this.